runtime/onert/backend/train/BackendContext.cc

```diff
@@ -103,11 +127,52 @@ backend::train::ITensorRegistry *BackendContext::genTrainingTensors()
       tensor_builder->notifyBackwardFirstUse(ind);
   });
 
+  for (const auto &op_index : tgraph.btopolSortOperations())
```

Comment:
I have another little question. 

Is it possible to replace `tgraph.btopolSorrtOperations()` with an operation vector(`backward_order`) that applied `truncatedBackwardOrder`? 

https://github.com/Samsung/ONE/blob/63e81d051bc0d01fe243238d843e450bfee46164/runtime/onert/core/src/compiler/ExecutorFactory.cc#L750-L753

I thought that if backward is not going to work, there is no need to allocate tensor related to that.

Reply:
BTW, This is a bit out of context. So, please proceed without considering this question.

Reply:
Thanks for your information. I missed out `truncateBackwardOrder`.

Reply:
@Aeren1564 Are you going to apply `truncateBackwardOrder` to all where `btopolSortOperations()` is being used? If so, Could you apply it to here as well?